### ISSUE TITLE
Add user confirmation to the apache configuration update script

### ISSUE THREAD

**Issue by miguelpduarte** [NIAEFEUP/niployments-old#5]
Title. See https://stackoverflow.com/questions/1885525/how-do-i-prompt-a-user-for-confirmation-in-bash-script

Not sure if this will be necessary (was not until now) but might be something that's nice to have.
